internal/dvstore/query.go

```diff
@@ -75,13 +76,30 @@ func LoadTasks(h *cayley.Handle, schema *schema.Config, filters LoadTasksFilters
 		paths = append(paths, path.StartPath(h))
 	} else {
 		for _, target := range filters.Targets {
-			// FIXME: handle different target types (for now only repo)
-			p := path.StartPath(h, quad.IRI(target.String())).
-				Both().
-				Has(quad.IRI("rdf:type"), quad.IRI("dv:Task"))
-
-			// FIXME: reverse depends/blocks
-			paths = append(paths, p)
+			switch target.Provider() {
+			case multipmuri.GitHubProvider:
+				// FIXME: handle different target types (for now only repo)
+				p := path.StartPath(h, quad.IRI(target.String())).
+					Both().
+					Has(quad.IRI("rdf:type"), quad.IRI("dv:Task"))
+
+				// FIXME: reverse depends/blocks
+				paths = append(paths, p)
+			
+			case multipmuri.TrelloProvider:
+				cardsId, err := trello.GetCardsId(target.LocalID()[1:], trelloToken, trelloApiKey)
```

Comment:
Seem's weird to me to use Trello API to get informations for retrieve data on local store.
Is it the only way ?

Reply:
Yep same, but once again, it may come from a design limitation that I introduced before.

Maybe it makes sense to support lazy fetches at some points.